cmd/nodecmd/validatePrimary.go

```diff
@@ -220,53 +221,71 @@ func promptWeightPrimaryNetwork(network models.Network) (uint64, error) {
 	}
 }
 
-func getTimeParametersPrimaryNetwork(network models.Network) (time.Time, time.Duration, error) {
+func getTimeParametersPrimaryNetwork(network models.Network, nodeIndex int) (time.Time, time.Duration, error) {
 	const (
 		defaultDurationOption = "Minimum staking duration on primary network"
 		custom                = "Custom"
 	)
+	var err error
 	start := time.Now().Add(constants.PrimaryNetworkValidatingStartLeadTime)
-	if duration == 0 {
-		msg := "How long should your validator validate for?"
-		durationOptions := []string{defaultDurationOption, custom}
-		durationOption, err := app.Prompt.CaptureList(msg, durationOptions)
+	if useCustomDuration && duration != 0 {
+		return start, duration, nil
+	}
+	if duration != 0 {
+		duration, err = getDefaultValidationTime(start, network, nodeIndex)
 		if err != nil {
 			return time.Time{}, 0, err
 		}
-
-		switch durationOption {
-		case defaultDurationOption:
-			duration, err = getDefaultMaxValidationTime(start, network)
-			if err != nil {
-				return time.Time{}, 0, err
-			}
-		default:
-			duration, err = subnetcmd.PromptDuration(start, network)
-			if err != nil {
-				return time.Time{}, 0, err
-			}
+		return start, duration, nil
+	}
+	msg := "How long should your validator validate for?"
+	durationOptions := []string{defaultDurationOption, custom}
+	durationOption, err := app.Prompt.CaptureList(msg, durationOptions)
+	if err != nil {
+		return time.Time{}, 0, err
+	}
+	switch durationOption {
+	case defaultDurationOption:
+		duration, err = getDefaultValidationTime(start, network, nodeIndex)
+		if err != nil {
+			return time.Time{}, 0, err
+		}
+	default:
+		useCustomDuration = true
+		duration, err = subnetcmd.PromptDuration(start, network)
+		if err != nil {
+			return time.Time{}, 0, err
 		}
 	}
 	return start, duration, nil
 }
 
-func getDefaultMaxValidationTime(start time.Time, network models.Network) (time.Duration, error) {
+func getDefaultValidationTime(start time.Time, network models.Network, nodeIndex int) (time.Duration, error) {
 	durationStr := constants.DefaultFujiStakeDuration
 	if network == models.Mainnet {
 		durationStr = constants.DefaultMainnetStakeDuration
 	}
-	d, err := time.ParseDuration(durationStr)
+	durationInt, err := strconv.Atoi(durationStr[:len(durationStr)-1])
 	if err != nil {
 		return 0, err
 	}
-	end := start.Add(d)
-	confirm := fmt.Sprintf("Your validator will finish staking by %s", end.Format(constants.TimeParseLayout))
-	yes, err := app.Prompt.CaptureYesNo(confirm)
+	// stagger expiration time by 1 day for each added node
+	durationAddition := 24 * nodeIndex
+	durationStr = strconv.Itoa(durationInt+durationAddition) + "h"
```

Comment:
You may want to better clarify that the validation period gets longer per node you add in bulk?

Reply:
I think that is a better option than setting the start to be delayed, but we may want to log that